taxi/models.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):

```

Comment:
You should implement the `Meta` class only after you have defined all the attributes. According to the checklist, the `Meta` class should be implemented after all attributes are defined, but here it's missing. Please, add it.